README.md

```diff
@@ -1,42 +1,31 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+Mobile App VERSION
+Project Name
+MOBILE-VERSION
 
-# Project Name
+ Project Description.
 
->my-portfolio  
+Built With
+Major languages HTML and CSS
+Frameworks
+Stylelint e.t.c
 
+👤 Author1
+ADEBOWALE ADEGBOYE
+GitHub: @ademibowale
+Twitter: @ademibowale1
+LinkedIn: https://www.linkedin.com/in/adebowale-adegboye-143568221/
 
-## Built With
-
-- Major languages HTML and CSS
-- Frameworks
-- Stylelint e.t.c
-
-## Authors
-
-👤 *Author1*
-
-- GitHub: [@adebowale](https://github.com/adebowale)
-- Twitter: [@adebowale](https://twitter.com/adebowale)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/adebowale)
-
```

Comment:
- [ ] please add these social media accounts as the previous reviewer didn't require to delete them